### PR TITLE
Console 3155: Display Operator logos on a white background on dark them to prevent visual issues and consistency

### PR DESCRIPTION
**Pull request by sg00dwin**
Addresses: https://issues.redhat.com/browse/CONSOLE-3155

Effected screens
```
/operatorhub/ns/default
/operatorhub/ns/default?details-item=...
/operatorhub/subscribe?pkg=...
/k8s/all-namespaces/operators.coreos.com~v1alpha1~ClusterServiceVersion
/catalog/ns/default
/catalog/source-to-image?imagestream=...
/catalog/instantiate-template?template=...
/topology/ns/EMPTY_PROJECT quick search (start building your application)
```

Screen Shot 2022-05-12 at 3 41 16 PM
Screen Shot 2022-05-12 at 3 40 35 PM
Screen Shot 2022-05-12 at 3 39 58 PM
Screen Shot 2022-05-12 at 3 39 04 PM
Screen Shot 2022-05-12 at 3 38 33 PM
Screen Shot 2022-05-12 at 3 37 37 PM

### CODE REVIEW

**Review by vikram-raj — COMMENTED** [review]
I noticed a few issues.
- size catalog image has been increased with this change

**This PR**
image

**Master**
image

- in dark mode operatorhub images look fine but in light mode it aligned to right. not sure if it is expected.
image

**Comment by sg00dwin** [comment]
> * in dark mode operatorhub images look fine but in light mode it aligned to right. not sure if it is expected
@vikram-raj I've updated the pr changes so that the logos `` in light theme are left edge aligned. This correctly aligns most. And ideally the logo is cropped at the very edge of the brand mark. But there are some `` files that have extra spacing between the logo itself and the edge of the `` so they will not look aligned, but technically the it is. Fortanix is a good example.  
logo-extra-spacing

**Comment by sg00dwin** [comment]
/label docs-approved
/label px-approved

We have docs- and px-approval for the epic.

QE Approver:
/assign @yanpzhan

File: frontend/public/style/_overrides.scss
Comment (mattnolting) [inline frontend/public/style/_overrides.scss:18]: Might consider bumping this to around 100, some of the logos are a bit hard to see.

**Review by mattnolting — APPROVED** [review]
This looks great! Some of the logos are on the small side (Citrix, CloudBees, CrowdStrike, Instana), might bump the `max-width`.

**Review by jcaianirh — APPROVED** [review]
/lgtm

went through e2e operator install flow in the admin perspective and the add pages of the dev perspective

**Comment by sg00dwin** [comment]
@vikram-raj can you give this approval thanks

**Comment by jerolimov** [comment]
Tested admin (operator hub, installed operators) and dev perspective (catalog) locally and it works as expected. :+1:

/approve

**Pull request merged**
